test/conformance/kuttl/rbac/aggregate-to-admin/admin-generaterequest.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+  name: kyverno:admin-generaterequest
+rules:
+- apiGroups:
+  - kyverno.io
+  resources:
+  - generaterequests
```

Comment:
We no longer create generaterequests. Do you mean updaterequests?

Reply:
Updaterequests is added in `test/conformance/kuttl/rbac/aggregate-to-admin/admin-updaterequest.yaml` , so i think we don't need this anymore

Reply:
I took the cluster roles from an actual deployment, either the test is wrong or the resources we create are wrong.